_posts/en/newsletters/2020-04-08-newsletter.md

```diff
@@ -0,0 +1,135 @@
+---
+title: 'Bitcoin Optech Newsletter #92'
+permalink: /en/newsletters/2020/04/08/
+name: 2020-04-08-newsletter
+slug: 2020-04-08-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter contains a list of notable changes to popular
+Bitcoin infrastructure projects.  FIXME: mention_PR_club_section_if_added
+
+## Action items
+
+*None this week.*
+
+## News
+
+*No significant development news this week.*
+
+## Bitcoin Core PR Review Club
+
+_In this section, we summarize a recent Bitcoin Core PR Review Club meeting,
+highlighting some of the important questions and answers.  Click on a
+question below to see a summary of the answer from the meeting._
+
+FIXME:jnewbery (write or delete section)
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[libsecp256k1][libsecp256k1 repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [C-Lightning #3612][] adds startup parameters `--large-channels` and
+  `--wumbo` (which are equivalent).  If used, the node will advertise
+  support for `option_support_large_channel` in its `init` message,
+  meaning it will accept channel opens with a value higher than the
+  previous limit of about 0.168 BTC.  If the remote peer also supports
+  this option, C-Lightning's `fundchannel` RPC will allow the user to
+  create channels over the previous limit.  See also Eclair's support
+  for this option described in [Newsletter #88][news88 eclair1323].
+
+- [C-Lightning #3600][] adds experimental support for *onion messages*
+  using *blinded paths*:
+
+    - *Onion messages* (called "LN direct messages" in
+      [Newsletter #86][news86 ln dm]) allow a node to send an encrypted
+      message across the network without using the LN payments
+      mechanism.  This can replace the mechanism of
+      messages-over-payments used by apps such as [Whatsat][].  Compared
+      to messages-over-payments, onion messages have several advantages:
```

Comment:
I think a large advantage is that onion messages don't use up channel liquidity while they're in flight. Should that be added to this list?

Reply:
Thought about this some and I think this could also be a disadvantage in that paid messages also pay fees and so provide an incentive for delivery.  E.g., WhatSat's readme [says](https://github.com/joostjager/whatsat#introduction), "Ideally users would [...] drop off fees along the way."  For that reason, I'm going to leave it out of the list.